Dockerfile.proxy

```diff
@@ -0,0 +1,10 @@
+FROM unleashorg/unleash-proxy
+
+ENV PORT 4242
+EXPOSE 4242
+
+COPY ./scripts/load_secrets_and_run.sh load_secrets_and_run.sh
+
+ENTRYPOINT ["/usr/local/bin/docker-entrypoint.sh", "./load_secrets_and_run.sh"]
+
+CMD ["/bin/sh" "-c" "./server.sh"]
```

Comment:
Got this CMD from the [image layers](https://hub.docker.com/layers/unleashorg/unleash-proxy/latest/images/sha256-748fc3ae83ed3883d5b53ef21ab943f5c7c54291821aaf0fe61e5a2f835df7fd?context=explore)